### DISCUSSION TITLE
Offline Mode

### DISCUSSION THREAD

**Discussion by mattdesl** [rainbow-me/rainbowkit#608] [Ideas]
Hey, I'd like to use a Connect button to help users send a signed message without requiring any connection to Infura, Alchemy, or a public provider. Is this possible with Rainbowkit?

**Comment by saihaj** [comment]
Signing a message in itself is done by wallet where you don’t need to connect to a Node but the wallet itself connects to a node and that’s where a Node connection is needed. I don’t think this is something rainbowkit can handle.
